specification/machinelearningservices/AzureAI.Assets/routes.tsp

```diff
@@ -120,3 +134,118 @@ interface Indexes {
     PagedIndex
   >;
 }
+
+@added(Versions.`2024-05-01-preview`)
+interface Prompts {
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "Waiting for bug fix: https://github.com/Azure/typespec-azure-pr/issues/3739."
```

Comment:
Are you copy-paste these operations? That issue in "suppress" is resolved. You should be unblocked.

Reply:
thank you for letting me know. Given that this is a rather large rewrite at this juncture, could we incorporate this feedback for the next API? thank you!

Reply:
followed-up: acceptable to do a follow-up PR since this will not require a new API version